Makefile

```diff
@@ -238,6 +238,10 @@ bin/containerd-shim-runc-v2: cmd/containerd-shim-runc-v2 FORCE # set !cgo and om
 	@echo "$(WHALE) $@"
 	@CGO_ENABLED=${SHIM_CGO_ENABLED} $(GO) build ${GO_BUILD_FLAGS} -o $@ ${SHIM_GO_LDFLAGS} ${GO_TAGS} ./cmd/containerd-shim-runc-v2
 
+# Build the windows shim if needed
+bin/containerd-shim-runhcs-v1.exe: script/setup/install-runhcs-shim go.mod
```

Comment:
Hmm, can't we keep Windows stuff on Makefile.windows? What would be a blocker?

Reply:
Other than my crappy make knowledge, probably nothing 😆 It also made sense to me to place all of the shims near each other and just let GOOS decide what extra to build.

Reply:
Fixed this by using .DEFAULT_GOAL := all as discussed below